packages/drupal/gutenberg_blocks/js/blocks/teaser-list.tsx

```diff
@@ -27,15 +27,18 @@ registerBlockType<{
     },
     buttonText: {
       type: 'string',
+      default: '',
     },
     contentHubEnabled: {
       type: 'boolean',
     },
     limit: {
-      type: 'string',
+      type: 'number',
```

Comment:
This is a breaking change I'm afraid. No idea how existing blocks will behave after the type change. Have you checked?

Reply:
As tested locally:
- if the field for some reason had a text string like `test`, then the field gets set to the number `0`
- backend editor works as expected

I've put `test` in the Limit for one node on `prod` so we can see if it breaks that build when we merge this, but it should work as expected